package.json

```diff
@@ -85,7 +85,7 @@
     "husky": "^9.0.7",
     "postcss": "^8",
     "prettier": "^3.2.4",
-    "prisma": "^5.6.0",
+    "prisma": "^5.12.1",
```

Comment:
why dep version changes?

Reply:
I had some error related to Prisma so I just installed Prisma manually, any problem?